docs/rst/tutorials/core/qos/xml_profiles/xml_profiles.rst

```diff
@@ -0,0 +1,400 @@
+.. include:: ../../../../exports/alias.include
+
+.. _tutorials_xml_profiles_intro:
+
+Configuring Fast-DDS QoS via XML profiles
+=========================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+XML configuration
+-----------------
+
+Fast DDS over Vulcanexus offers the possibility of fully configuring QoS policy through XML profile definition.
+This can be done in several ways depending on whether the application used is a native Fast DDS application or a ROS 2 application
+using Fast DDS's RMW implementation.
+This guide will focus on the latter.
+For more information regarding XML profiles in Fast DDS please refer to the
+`XML configuration page <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html>`_
+from Fast DDS's documentation.
+
+.. _vulcanexus_configure:
+
+Configuring *Fast DDS* in Vulcanexus for ROS 2 applications
+-----------------------------------------------------------
+
+ROS 2 only allows for the configuration of certain middleware QoS
+(see `ROS 2 QoS policies <https://index.ros.org/doc/ros2/Concepts/About-Quality-of-Service-Settings/#qos-policies>`_).
+However, *rmw_fastrtps* offers extended configuration capabilities
+to take full advantage of the features in *Fast DDS*.
+This section describes how to specify this extended configuration.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 2
+
+.. _vulcanexus_configure_publication_mode:
+
+Changing publication mode
+-------------------------
+
+*rmw_fastrtps* in ROS 2 uses asynchronous publication by default.
+This can be easily changed setting the environment variable ``RMW_FASTRTPS_PUBLICATION_MODE``
```

Comment:
Please avoid qualitative adjectives like this one
```suggestion
This can be changed setting the environment variable ``RMW_FASTRTPS_PUBLICATION_MODE``
```